README.md

```diff
@@ -2,26 +2,54 @@
 
 ![Screenshot from 2023-03-18 15-29-05](https://user-images.githubusercontent.com/8228060/226112255-fe2d4cdc-193e-4c23-8a36-3d8f60baaf03.png)
 
+This project contains an awesome application called `QuickPizza` and also demonstrates how to use the different features of k6. 
+
+You'll find sample tests about:
```

Comment:
Thoughts on being a bit less concrete in here? Like, saying that it covers k6's foundations, with a few words inside the parenthesis, and also X, Y, Z. With the current list, I feel bad about not including all the things that are doable :smile:

Reply:
Have changed this to be more general: `It also demonstrates the basic and advanced functionalities of k6, ranging from a basic load test to using different modules and extensions.`